### DISCUSSION TITLE
脚注与正文的间距不固定

### DISCUSSION THREAD

**Discussion by EleLawrence** [obster-y/XJTU-thesis#71] [Q&A]
尊敬的作者，

我在使用该模板时发现，脚注和正文的间距经常变化。下面展示了三个例子。

**例一：摘要的脚注**

![摘要](https://user-images.githubusercontent.com/54851956/170548317-d4d80392-3234-4dd1-a3ac-3d545a02041a.png)

图中，脚注和摘要正文之间的间距感觉较为合理。

**例二：正文的脚注一**

![正文1](https://user-images.githubusercontent.com/54851956/170548642-a96b1d2b-00cf-448e-913c-34627d69f13b.png)

可见，图中脚注和正文的间距较大（我论文中的多个脚注都出现了这样的情况）。

**例三：正文的脚注二**

![正文2](https://user-images.githubusercontent.com/54851956/170548478-d910379c-b7d1-4112-844a-1c13a6e93a71.png)

图中，脚注和正文的间距似乎又太窄。

我不确定以上现象是否是正常的，或是我没有正确使用模板。希望得到您的答复，非常感谢！

备注：
模板版本：DEV分支5月25日版本
编译环境：Windows，用VS Code编译

**Comment by sikouhjw** [comment]
这个间距似乎是弹性距离，也就是说距离不固定，根据本页内容去调整，建议不管咯。

**Reply by EleLawrence** [reply]
原来如此，那我就不管它了。感谢！
